puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/foreman-release-pipelines.yaml

```diff
@@ -52,12 +52,28 @@
         - 'pipelines/lib/release.groovy{empty}'
         - 'pipelines/lib/foreman_infra.groovy{empty}'
 
+- job-template:
+    name: 'foreman-plugins-{version}-test-pipeline'
+    project-type: pipeline
+    sandbox: true
+    dsl:
+      !include-raw:
+        - 'pipelines/vars/foreman/{version}.groovy'
+        - 'pipelines/release/pipelines/plugins-pipeline.groovy{empty}'
+        - 'pipelines/lib/rvm.groovy{empty}'
+        - 'pipelines/lib/ansible.groovy{empty}'
+        - 'pipelines/lib/obal.groovy{empty}'
+        - 'pipelines/lib/rpm_packaging.groovy{empty}'
+        - 'pipelines/lib/release.groovy{empty}'
```

Comment:
I think that for these only `ansible.groovy` is needed since it only calls `runCicoPipelines` (which depends on `runPlaybook`)